### PR TITLE
Remove confirm passphrase and/or password prompt when decrypting and signing

### LINKED ISSUE #8072: Remove confirm passphrase and/or password prompt when decrypting and signing
Encrypting should always prompt user to repeat passphrase and/or password.
Decrypting and signing should never prompt user to repeat passphrase and/or password.

### PR DESCRIPTION
**Pull request by bobanm**
### What was the problem?

This PR resolves #8072

### How was it solved?

Removed `shouldConfirm` argument from `getPassphraseFromPrompt()` and `getPasswordFromPrompt()` for applicable commands.

The commands which no longer require repeat passphrase are:
* `transaction:create`
* `transaction:sign`
* `message:sign`

The commands which no longer require repeat password are:
* `passphrase:decrypt`

### How was it tested?

All tests OK 👌🏻

### CODE REVIEW

File: commander/src/bootstrapping/commands/keys/encrypt.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ export class EncryptCommand extends Command {
 
 	async run(): Promise<void> {
 		const { flags } = await this.parse(EncryptCommand);
-		const password = flags.password ?? (await getPasswordFromPrompt('password', true));
+		const password = flags.password ?? (await getPasswordFromPrompt('password'));
```
Comment (shuse2) [inline commander/src/bootstrapping/commands/keys/encrypt.ts:79]: As discussed offline, when `encrypt` we should confirm the password

File: commander/src/bootstrapping/commands/passphrase/encrypt.ts [hunk 2]
```diff
@@ -58,7 +58,7 @@ export class EncryptCommand extends Command {
 			fs.ensureDirSync(dir);
 		}
 
-		const passphrase = passphraseSource ?? (await getPassphraseFromPrompt('passphrase', true));
+		const passphrase = passphraseSource ?? (await getPassphraseFromPrompt('passphrase'));
```
Comment (shuse2) [inline commander/src/bootstrapping/commands/passphrase/encrypt.ts:61]: same as above

File: commander/src/commands/message/encrypt.ts [hunk 3]
```diff
@@ -80,7 +80,7 @@ export default class EncryptCommand extends BaseCommand {
 		if (!message && !messageSource) {
 			throw new ValidationError('No message was provided.');
 		}
-		const passphrase = passphraseSource ?? (await getPassphraseFromPrompt('passphrase', true));
+		const passphrase = passphraseSource ?? (await getPassphraseFromPrompt('passphrase'));
```
Comment (shuse2) [inline commander/src/commands/message/encrypt.ts:83]: same as above

**Review by shuse2 — CHANGES_REQUESTED** [review]

**Review by mosmartin — APPROVED** [review]
🚀

**Review by shuse2 — APPROVED** [review]

**Pull request merged**